index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div>
+      <h1 id ="title">Microverse Survey</h1>
+      <p id = "description">Fill with your data</p>
+    </div>
+
+    <form id ="survey-form">
+
+ 
+          <label id ="name-label">Name</label>
+          <input placeholder="Enter your name" id="name"  type ="text" name="name" required>
+ 
+
+          <label id ="lastName-label">Last name</label>
+          <input placeholder="Enter your last name" type ="text" name="lastName" id="lastName">
+
+
+
+          <label id="email-label">Email</label>
+          <input  placeholder ="Enter your email" id="email" type="email" name="email" required>
+
+
```

Comment:
NIce code right there! You should avoid using blank spaces between your code lines to make it cleaner and easier to read.